### PR TITLE
[MINOR] reorder the comments for MasterCount pruning in master.go

### PR DESCRIPTION
**Pull request by jayunit100**
The way master count deletes things is a little  tricky.  Figured these comments make it a little easier to interpret.

### CODE REVIEW

File: pkg/master/controller.go [hunk 1]
```diff
@@ -212,18 +212,22 @@ func (c *Controller) SetEndpoints(serviceName string, ip net.IP, port int) error
 		glog.Warningf("Resetting endpoints for master service %q to %v", serviceName, e)
 		return c.EndpointRegistry.UpdateEndpoints(ctx, e)
 	} else if !ipCorrect {
-		// We *always* add our own IP address; if there are too many IP
-		// addresses, we remove the ones lexicographically after our
+		// We *always* add our own IP address;
```
Comment (pmorie) [inline pkg/master/controller.go]: Nit, the `;` is no longer necessary on this line.

**Comment by pmorie** [comment]
LGTM modulo nit

**Comment by jayunit100** [comment]
okay, rebased .thanks for the review

**Comment by jayunit100** [comment]
@k8s-bot test this please

**Comment by jayunit100** [comment]
looks like a shippable flake. this is  just comments so should be good to merge i guess...

**Pull request merged**